src/openforms/formio/rendering/default.py

```diff
@@ -345,3 +350,60 @@ def is_visible(self) -> bool:
         registration data...
         """
         return False
+
+
+@register("partners")
+class PartnersNode(ComponentNode):
+    @property
+    def value(self) -> Any:
+        value = glom(self.step_data.data, self.path or self.key)
```

Comment:
`self.step_data` is of type `FormioData` and `glom` has serious performance issues, instead you should be able to simply do:

```py
value = self.step_data[self.path or self.key]
```
thanks to @viktorvanwijk 's efforts :)